Source/AwsCommonRuntimeKit/crt/CommandLine.swift

```diff
@@ -3,7 +3,8 @@
 
 import Foundation
 import AwsCCommon
-//swiftlint:disable trailing_whitespace
+// swiftlint:disable trailing_whitespace
+// Todo: remove/refactor
 public struct CommandLineParser {
```

Comment:
Will refactor it in future to use https://apple.github.io/swift-argument-parser/documentation/argumentparser/.